xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2156,6 +2156,7 @@ void CVideoPlayer::CheckContinuity(CCurrentStream& current, DemuxPacket* pPacket
        fabs(this_dts - that_dts) < DVD_MSEC_TO_TIME(1000))
     {
       m_offset_pts += correction;
+      m_VideoPlayerVideo->SetOffsetPts(m_offset_pts);
```

Comment:
> all demuxer packets including subs are corrected by UpdateTimestamps. UpdateTimestamps is before PTSStartTIme is set.

that's true but UpdateTimestamps doesn't correct the start / stop of subtitles. That ONLY seems to happen in https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/VideoPlayerSubtitle.cpp#L213 and if you look, this is only for a subtitle FILE parser. I may be wrong but I searched the entire code for those start/stop corrections and it's the only spot I could find...